pkg/components/components.go

```diff
@@ -116,7 +116,10 @@ func GetDeploymentSpecOperator(namespace, image, imagePullPolicy, conversionCont
 		Template: corev1.PodTemplateSpec{
 			ObjectMeta: metav1.ObjectMeta{
 				Labels: map[string]string{
-					"name": hcoName,
+					"name":                    hcoName,
+					hcoutil.AppVersionLabel:   hcoKvIoVersion,
```

Comment:
Here you add the labels directly to the podSpec - shouldn't this be added to the Deployment's labels too?
Same comment applies both to the operator and webhook deployments.

Reply:
I discussed this with @tiraboschi  and explained that OLM doesn't allow us to set labels for deployments. It takes only DeploymentSpec.

Reply:
Seems like this capability was recently added in [operator-framework/api#46](https://github.com/operator-framework/api/pull/46) (although [its parent issue](https://github.com/operator-framework/operator-lifecycle-manager/issues/1522) is still open).

Reply:
I added the labels into the StrategyDeploymentSpec objects as well. OLM is not currently using them but my understanding is that it is going to use.